troncli/cli.py

```diff
@@ -13,78 +13,72 @@
 @cbox.cmd
 def init(version: str):
     """Init dirs and fetch code.
-    >> 
-    Settings: 
-        --version
-    >>
+
+    Examples:
```

Comment:
this will be output as help information by cbox in the command line, let's try to make it consistent